src/main/resources/templates/team_page_apply_team.html

```diff
@@ -47,7 +48,7 @@ <h2  class="team-heading team-heading-current-team" th:text="#{teamPageApply.hea
                         <div class="form-group">
                             <div class="checkbox">
                                 <label><input type="checkbox" th:field="*{hasAcceptTeamOwnerPolicy}"/> <a href="/team_owner_policy" th:src="@{/team_owner_policy}"> <span th:text="#{signup.acceptTeamOwnerPolicy}">I accept the team owner policy</span></a></label>
-                                <span class="text-danger" th:if="${#fields.hasErrors('hasAcceptTeamOwnerPolicy')}" th:errors="*{hasAcceptTeamOwnerPolicy}">Team owner error</span>
+                                <span class="account-change-fail" th:if="${#fields.hasErrors('hasAcceptTeamOwnerPolicy')}"><br/><i class="fa fa-exclamation-circle"></i> <span th:errors="*{hasAcceptTeamOwnerPolicy}"></span></span>
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this <i> tag by <em>. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)